samples/snippets/fsharp/VS_Snippets_CLR_System/system.convert.tostring2/fs/tostring2.fs

```diff
@@ -0,0 +1,172 @@
+open System
+
+let convertByte =
```

Comment:
Just wasn't sure what's the best way to convert a C# code with side-effects that prints to the console. There are three options I see: 
1. make it pure (the functional way) but print in the main
2. keep it as close to C# as possible with side effects ignoring the functional specific i.e. make it (unit -> unit) or just (unit) both are not ideal.